src/AWS/Orleans.Clustering.DynamoDB/Membership/DynamoDBGatewayListProvider.cs

```diff
@@ -35,13 +35,9 @@ internal class DynamoDBGatewayListProvider : IGatewayListProvider
 
         public Task InitializeGatewayListProvider()
         {
-            this.storage = new DynamoDBStorage(
-                this.logger,
-                this.options.Service,
-                this.options.AccessKey,
-                this.options.SecretKey,
-                this.options.ReadCapacityUnits,
-                this.options.WriteCapacityUnits);
+            this.storage = new DynamoDBStorage(this.logger, this.options.Service, this.options.AccessKey, this.options.SecretKey,
```

Comment:
That makes sense. I've changed it so that it's consistent with the other usages of the `DynamoDBStorage` class.
I'll change back in a future PR.

Thanks for reviewing and merging @ReubenBond.